app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/CheckAgencyController.scala

```diff
@@ -59,13 +59,8 @@ class CheckAgencyController @Inject()
   }
 
   val showCheckAgencyStatus: Action[AnyContent] = AuthorisedWithSubscribingAgent {
```

Comment:
W e have to figure out the best possible impl, at the same time ruling out unnecessary auth call because:
- it makes additional load to auth
- it takes additional time